recipes/pocketsphinx/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "pocketsphinx" %}
+{% set version = "0.8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://sourceforge.net/projects/cmusphinx/files/{{ name }}/5prealpha/{{ name }}-5prealpha.tar.gz
+  sha256: ef5bb5547e2712bdf571f256490ef42a47962033892efd9d7df8eed7fe573ed9
+
+build:
+  skip: True  # [win or osx]
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - automake
+    - libtool
+    - swig
+    - sphinxbase
+    - pkg-config
+    - gcc
```

Comment:
hmm you shouldn't need to have both gcc and compiler('c')

Reply:
Thanks for noticing that.  I have removed `gcc`.